plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx

```diff
@@ -69,6 +71,9 @@ const CreateEditChronogramTaskModal: FunctionComponent<Props> = ({
     const allowConfirm =
         !formik.isSubmitting && formik.isValid && isFormChanged;
 
+    const currentUser = useCurrentUser();
+    const isUserAdmin = userHasPermission('iaso_polio_chronogram', currentUser);
```

Comment:
The name of the isUserAdmin should be changed for example to UserWriteRead because isUserAdmin can make a confusion with the admin one

Reply:
OK

Reply:
Fixed in https://github.com/BLSQ/iaso/pull/1530/commits/f0cc7762dfe8ba1d92621cee05a6bf1204ba74ad